src/styles/blocks/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (min-width: 768px) and (max-width: 1023px) {
+  .cards {
+    grid-template-columns: repeat(3, 200px) !important;
```

Comment:
без !important не работает

Reply:
Please contact the chat, because you have a project from scratch, and if it doesn't work without important, you have a bug somewhere and you interrupt your styles yourself. It shouldn't be like that, you don't have external libraries that could interfere with you. It is necessary to rewrite the styles and not just delete the important and say that it does not work.